client/src/components/ActionMenu/ActionMenu.js

```diff
@@ -25,45 +31,32 @@ class ActionMenu extends PureComponent {
     //
     // ignore linting rule because following the recommended replacement caused behat failures
     // eslint-disable-next-line react/no-access-state-in-setstate
-    window.setTimeout(() => this.setState({ isOpen: !this.state.isOpen }), 0);
```

Comment:
Might be worth checking this comment is still valid now that we're using the `useState` hook